### PR TITLE
Documentation for get cdnsettings

### PR DESCRIPTION
**Pull request by evanluc**
Adding documentation for getting cdn settings.

### CODE REVIEW

File: source/includes/stackpath/_cdn_settings.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+`queryStringControl`<br/>*String* | The strategy for caching query strings. Can be `IGNORE`, `CACHE_ALL` or `CUSTOM`. 
+`customCachedQueryStrings`<br/>*Array[String]* | List of custom cached query strings. Only visible if the `queryStringControl` attribute is `CUSTOM`
+`dynamicCachingByHeaderEnabled`<br/>*Boolean* | Whether or not to enable dynamic caching by headers.
+`customCachedHeaders`<br/>*Array[String]*  | A list of custom cached headers. Only visible if `dynamicCachingByHeaderEnabled` is enabled. 
+`gzipCompressionEnabled`<br/>*Boolean* | Whether or not to enable gzip compression.
+`gzipCompressionLevel`<br/>*Integer* | The level for the gzip compression. Values are between `1` to `6`. Only visible is `gzipCompressionEnabled` is enabled.
+`contentPersistenceEnabled`<br/>*Boolean* | Whether or not make cached content available after its expiration time.
+`maximumStaleFileTtl`<br/>*Integer*  | The maximum time to live for stale files. Only visible if `contentPersistenceEnabled` is enabled. 
+`varyHeaderEnabled`<br/>*Boolean* | Whether or not to enable honor the vary header in a request.
```
Comment (GuillaumeVialle) [inline source/includes/stackpath/_cdn_settings.md]: is this correct? `to enable honor the vary header` should not be `to enable the vary header`?

**Review by GuillaumeVialle — APPROVED** [review]

File: source/includes/stackpath/_cdn_settings.md (hunk 1 above)
Comment (evanluc) [reply source/includes/stackpath/_cdn_settings.md]: Yup! I got it from here: 
image

Comment (evanluc) [inline source/includes/stackpath/_cdn_settings.md]:
```suggestion
`varyHeaderEnabled`<br/>*Boolean* | Whether or not to enable honoring the vary header in a request.
```

Comment (evanluc) [reply source/includes/stackpath/_cdn_settings.md]: https://github.com/cloudops/cloudmc-api-docs/pull/326/commits/a8bb75a6b0a5b5efcfdef89b122363e6f6adde03 did fix small typo here though ahah

File: source/includes/stackpath/_cdn_settings.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+`maximumStaleFileTtl`<br/>*Integer*  | The maximum time to live for stale files. Only visible if `contentPersistenceEnabled` is enabled. 
+`varyHeaderEnabled`<br/>*Boolean* | Whether or not to enable honoring the vary header in a request.
+`browserCacheTtl`<br/>*Integer*  | Sets the default browser expiration time for cached assets. 
+`corsHeaderEnabled`<br/>*Boolean* | Sets the Access-Control-Allow-Origin header to allow browsers to access this domain from other origins.
+`allowedCorsOrigins`<br/>*String* | The strategy for allowing cors origins. Can be `SPECIFY_ORIGINS` or `ALL_ORIGINS`.
+`originsToAllowCors`<br/>*Array[String]* | A list of origins to allow cors requests from. Only visible if `allowedCorsOrigins` is set to `SPECIFY_ORIGINS`.
+`http2SupportEnabled`<br/>*Boolean* | Whether or not to enable supporting applications using HTTP/2 protocol.
+`http2ServerPushEnabled`<br/>*Boolean* | Whether or not to push assets to the client or browser (user) in advance (before the user requests these assets) which enables faster load times.
+`linkHeader`<br/>*String* | The link header for http2ServerPush, only visible is `http2ServerPushEnabled`<br/>*Boolean* is enabled.
```
Comment (RGitB) [inline source/includes/stackpath/_cdn_settings.md]:
```suggestion
`linkHeader`<br/>*String* | The link header for http2ServerPush, only visible if `http2ServerPushEnabled` is enabled. 
```

**Review by RGitB — APPROVED** [review]
one small change, otherwise LGTM

**Pull request merged**